packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -246,6 +246,7 @@ const makeRenderer = (
 					isProdApi: server.settings.app.api.isProd,
 					isQL: parseMemberCookie(state).ql === 'true',
 					memberId: parseMemberCookie(state).id,
+					memberState: parseMemberCookie(state).state,
```

Comment:
is this literally the state of the country that the member is in? With no PR description and no ticket, we don't have context for understanding the purpose of adding this property, and I have concerns about the maintainability of incrementally adding member cookie properties over time. We should consider loading all member cookie params into a `member` object here.

Please document!

Reply:
Absolutely I'll document some stuff and pass it in via a member object. 

It is the state of the country the member is in for reference.

It's a longer discussion but i'll try giving a gist in a comment.

Reply:
I would also suggest renaming this, I would have initially thought at a first glance this referred to the state of the member vs members physical location